gpt_playground/settings.py

```diff
@@ -475,8 +475,10 @@
 }
 # text-neutral text-error (pipeline run table styles here for tailwind build)
 DJANGO_TABLES2_ROW_ATTRS = {
-    "class": "border-b border-gray-200 hover:bg-gray-100",
+    "class": "border-b border-gray-200 hover:bg-gray-100 clickable",
     "id": lambda record: f"record-{record.id}",
+    "data-redirect-url": lambda record: record.get_absolute_url() if hasattr(record, "get_absolute_url") else "",
+    "style": "cursor: pointer",
```

Comment:
What if you added this in CSS to the clickable class? That way those items without a URL wouldn't look like they were clickable (if you also take the suggestion in my comment below)

Reply:
Good idea :+1:

Reply:
I used a class from tailwind instead: https://github.com/dimagi/open-chat-studio/pull/342/commits/2b33cd1029bfc89105c834332af4b4c3e1f8b5dd